lib/matplotlib/tests/test_widgets.py

```diff
@@ -137,7 +135,7 @@ def test_rectangle_drag(ax, drag_from_anywhere, new_center):
     tool = widgets.RectangleSelector(ax, interactive=True,
                                      drag_from_anywhere=drag_from_anywhere)
     # Create rectangle
-    click_and_drag(tool, start=(0, 10), end=(100, 120))
+    click_and_drag(tool, start=(10, 10), end=(90, 120))
```

Comment:
The positions in the last part of this test are also outside both before and after rectangles, so just checking if those are still valid, or are they actually outside the _Axes_ just as these were (and thus not checking what was intended)?